source/ranges.tex

```diff
@@ -656,6 +657,29 @@
 type is integer-like.
 \end{note}
 
+\rSec2[range.prim.ssize]{\tcode{ranges::ssize}}
+
+\pnum
+The name \tcode{ssize} denotes
+a customization point object\iref{customization.point.object}.
+The expression \tcode{ranges::ssize(E)}
+for a subexpression \tcode{E} of type \tcode{T}
+is expression-equivalent to:
+
+\begin{itemize}
+\item
+If \tcode{range_difference_t<T>} has width less than \tcode{ptrdiff_t}:
+\begin{codeblock}
+static_cast<ptrdiff_t>(ranges::size(E)); // \ref{range.prim.size}
```

Comment:
Semicolons seem odd here given that this is intended to be an expression rather than a statement. I also think the cross-reference to the immediately preceding subclause adds no value. `If \tcode{range_difference_t<T>} has width less than \tcode{ptrdiff_t}, \tcode{static_cast<ptrdiff_t>(ranges::size(E))}.` would seem to be an improvement. (Ditto for the following case.)

Reply:
done as a separate commit